bin/taskcat-v9

```diff
@@ -1,14 +1,4 @@
 #!/usr/bin/env python
-# authors:
-# Tony Vattathil tonynv@amazon.com, avattathil@gmail.com
-# Santiago Cardenas <sancard@amazon.com>, <santiago[dot]cardenas[at]outlook[dot]com>
-# Shivansh Singh sshvans@amazon.com,
-# Jay McConnell, jmmccon@amazon.com
-"""
- Program License: Amazon License
- Python Class License: Apache 2.0
-"""
-
 from __future__ import print_function
```

Comment:
These future imports were needed to get python3 print unde python2 right? 

We should consider removing them as v9 has py3 as a requirement. If not in this or, then an issue to tackle it in a separate style pr?